plugincheck/settings/base.py

```diff
@@ -89,16 +82,36 @@
 USE_TZ = True
 
 
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.7/howto/static-files/
-
-STATIC_URL = '/static/'
-
-
 AUTHENTICATION_BACKENDS = (
    'plugincheck.base.auth.OnlyMozillaBackend',
 )
 
+BROWSERID_AUDIENCES = [
+    'http://localhost:8000',
+    'https://plugincheck-dev.herokuapp.com',
+    'http://plugincheck-dev.herokuapp.com',
+    'https://plugincheck.herokuapp.com']
```

Comment:
Given that `DEBUG = False`, I wouldn't include `localhost:8000` in this list or the `-dev*` items, either. This should be as restrictive as possible.